tests/ts_util.erl

```diff
@@ -486,9 +486,9 @@ assert_float(String, Exp, Got) -> assert(String, Exp, Got).
 
 assertf2([], []) -> pass;
 assertf2([H1 | T1], [H2 | T2]) ->
-    Diff = H1 - H2,
-    Av = (H1 + H2)/2,
-    if Diff/Av > ?DELTA -> fail;
+    Diff = abs(H1 - H2),
+    Delta = Diff/H2,
```

Comment:
This looks like it won't work properly for negative floats; `H2` should also be an absolute value (or just absolute value `Delta`).